providers/ftp/src/airflow/providers/ftp/operators/ftp.py

```diff
@@ -27,7 +27,13 @@
 from pathlib import Path
 from typing import Any
 
-from airflow.models import BaseOperator
+from airflow.providers.ftp.version_compat import AIRFLOW_V_3_0_PLUS
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.sdk import BaseOperator
+else:
+    from airflow.models.baseoperator import BaseOperator
```

Comment:
Could you move this to `version_compat` file @sunank200  and change the import:

```
if AIRFLOW_V_3_0_PLUS:
    from airflow.sdk import BaseOperator
else:
    from airflow.modelsimport BaseOperator  # type: ignore[no-redef]
```

Reply:
changed it.